modules/51_internalizeDamages/KotzWenzCPreg/postsolve.gms

```diff
@@ -0,0 +1,54 @@
+*** |  (C) 2006-2023 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+
+*** SOF ./modules/51_internalizeDamages/KotzWenzItr/postsolve.gms
+
+
+p51_sccLastItr(tall,regi) = p51_scc(tall,regi);
+
+
+p51_scc(tall,regi)$((tall.val ge 2020) and (tall.val le 2150)) = 1000 *
+*p51_welf(tall)**(-1) * 
```

Comment:
Sometimes, it is a bit hard for me to guess what is actually a comment and what part is multiplied...